libs/wallet/src/vega-transaction-dialog/vega-transaction-dialog.tsx

```diff
@@ -84,15 +92,119 @@ interface VegaDialogProps {
   transaction: VegaTxState;
   finalizedOrder: Order | null;
   title: string;
+  edit?: (body?: unknown) => Promise<unknown>;
+  editOrder?: Order | null;
 }
 
+interface VegaOrderEditDialogProps {
+  title: string;
+  order: Order | null;
+  edit: (body?: unknown) => Promise<unknown>;
+}
+
+interface FormFields {
+  entryPrice: string;
+}
+
+export const VegaOrderEditDialog = ({
+  order,
+  title,
+  edit,
+}: VegaOrderEditDialogProps) => {
+  const headerClassName = 'text-h5 font-bold text-black dark:text-white';
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm<FormFields>({
+    defaultValues: {
+      entryPrice: order?.price
+        ? addDecimalsFormatNumber(
```

Comment:
question (blocking): I'm not sure about formatting the number with commas here as we don't account for them when removing decimals for sending to the wallet. I think just addDecimals will be safer